vipul/Gallery_System/macro.html

```diff
@@ -0,0 +1,105 @@
+							<li><a href="wild.html">Wildlife</a></li>
+							<li><a href="landscape.html">Landscape</a></li>
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="category-wild">
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro.jpg">
+						<img src="images/macro.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro1.jpg">
+						<img src="images/macro1.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro2.jpg">
+						<img src="images/macro2.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro3.jpg">
+						<img src="images/macro3.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro4.jpg">
+						<img src="images/macro4.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro5.jpg">
+						<img src="images/macro5.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro6.jpg">
+						<img src="images/macro6.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro7.jpg">
+						<img src="images/macro7.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/macro8.jpg">
+						<img src="images/macro8.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro9.jpg">
+						<img src="images/macro9.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro.jpg">
+						<img src="images/macro.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/macro1.jpg">
+						<img src="images/macro1.jpg" alt="Picture of Macro Photography">
+					</a>
+			</div>
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer
bad code indentation

Reply:
i will do all the changes